### DISCUSSION TITLE
My benefits are not there after 3 days

### DISCUSSION THREAD

**Discussion by ghost** [community/community#112094] [GitHub Education] [answered]
### Hi everyone,

Hey everyone so i wanna ask how long the period is to wait until i unlock my benefits after i got approved,
i got approved on 8 Mar 2024 and now i dont have my benefits
![image](https://github.com/community/community/assets/123818929/7afcbb50-4455-4769-9e8e-c1fbc933d6db)

**Comment by probablynotnmp** [comment, marked as answer]
Hi Asnalino, It seems many are experiencing difficulties with GitHub Education benefits. I would recommend waiting a bit longer as the issue may resolve itself. Alternatively, you could reach out to GitHub support for assistance.

**Reply by ghost** [reply]
i already reached out to the support but there is the problem i get the answer from the ai message and if i send more messages than nobody answers haha but i wait thank you!

**Comment by SebastienXia** [comment]
Hi, @asnalino 
Don't worry you will have the scheduled benefits within 5 - 6 days after being approved.

> So you just have to wait

**Reply by ghost** [reply]
yep thanks

**Comment by ghost** [comment]
UPDATE: got it thanks for ur asnwers<3

**Reply by mger1608** [reply]
@asnalino Did you receive a verification email or notification that you're benefits were available or did you just have to continue checking the application status page?

I am mainly hoping to gain access to Copilot, and the Copilot FAQ indicates that once you have access to student benefits you should be able to automatically sign up without charge. If you see a page asking for payment info, it means you have not been approved yet, but I am not sure when that verification process will occur.

I was approved yesterday and am now waiting to receive access to benefits like you.

**Reply by ghost** [reply]
You need to wait lik 72 hours until you get the benefits

**Reply by kfofkf123** [reply]
Hi, I also got approved on the 15th march, but haven't received the benefits nor the emails yet, I'm hoping I get it soon.

**Reply by ghost** [reply]
u got it :)

**Reply by AyushXJain** [reply]
I got approved on 21s March still not received benefits

**Reply by kfofkf123** [reply]
Update: I received my benefits 4 days after I got accepted, just wait for 3-4 days and you'll receive it soon, good luck
